src/frequenz/sdk/microgrid/component/_component_data.py

```diff
@@ -130,17 +130,28 @@ class BatteryData(ComponentData):
     capacity: float
     """The capacity of the battery in Wh (Watt-hour)."""
 
-    power_lower_bound: float
+    power_inclusion_lower_bound: float
     """The maximum discharge power, in watts, represented in the passive sign
         convention. this will be a negative number, or zero if no discharging is
         possible.
     """
```

Comment:
These descriptions seem outdated or not super precise at least. I would add a more descriptive explanation of what inclusion and exclusion bounds are, probably copying it [from the microgrid API](https://github.com/frequenz-floss/frequenz-api-microgrid/blob/5bd9d3858abda6101d8a1f979bee8130390da83b/proto/frequenz/api/microgrid/microgrid.proto#L97-L180).

Reply:
I have updated the docs and added a link to the microgrid repo as well.  There are similar docs in the high level interface in the draft battery pool PR as well:  https://github.com/frequenz-floss/frequenz-sdk-python/pull/537/files#diff-c8758cd7dae69ff76ee96db23ab561b0af768bc42a2286845366f0e9833a2238R32-R40